client/webserver/site/src/js/locales.ts

```diff
@@ -315,7 +328,20 @@ export const enUS: Locale = {
   [ID_TRADING_TIER_UPDATED]: 'Trading Tier Updated',
   [ID_INVALID_TIER_VALUE]: 'Invalid tier value',
   [ID_INVALID_COMPS_VALUE]: 'Invalid comps value',
-  [ID_API_ERROR]: 'api error: {{ msg }}'
+  [ID_API_ERROR]: 'api error: {{ msg }}',
+  [ID_TX_TYPE_UNKNOWN]: 'Unknown',
+  [ID_TX_TYPE_SEND]: 'Send',
+  [ID_TX_TYPE_RECEIVE]: 'Receive',
+  [ID_TX_TYPE_SWAP]: 'Swap',
+  [ID_TX_TYPE_REDEEM]: 'Redeem',
+  [ID_TX_TYPE_REFUND]: 'Refund',
+  [ID_TX_TYPE_SPLIT]: 'Split',
+  [ID_TX_TYPE_CREATE_BOND]: 'Create Bond',
```

Comment:
Don't think the type strings need to be Initial Caps

Reply:
Do you mean both words or just the second?

Reply:
Just the second, "Create bond" etc